src/pages/proposal.tsx

```diff
@@ -224,10 +234,10 @@ export const Proposal: React.FC = () => {
 
   // set editor data
   useEffect(() => {
-    if (proposal && editor) {
+    if (proposal && editor && proposal.metadata.description) {
       editor.commands.setContent(
         // Default list of allowed tags and attributes - https://www.npmjs.com/package/sanitize-html#default-options
-        sanitizeHtml(proposal.metadata.description, {
+        sanitizeHtml(proposal.metadata.description!, {
```

Comment:
We're already checking that `proposal.metadata.description` is not null, right?
```suggestion
        sanitizeHtml(proposal.metadata.description, {
```

Reply:
7abb8407